src/main/resources/static/templates/PasswordResetVulnerability/LEVEL_9/SecurePasswordReset.css

```
@@ -0,0 +1,175 @@
+}
+
+.pr-detail-key {
+  font-weight: 600;
+  color: #1f5131;
+}
+
+.pr-detail-value {
+  color: #10311f;
+  text-align: right;
+  max-width: 65%;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)